server/graphql/general/src/queries/item_variant.rs

```diff
@@ -9,128 +9,64 @@ pub struct VariantNode {
 }
 
 #[derive(SimpleObject)]
-pub struct ItemVariantNode {
-    pub id: String, // item id
-    pub unit: String,
-    pub most_used: String,
+pub struct UnitVariantNode {
```

Comment:
Changes some fields, and really wanted to use reference data when doing mock api like this, so changed here to match item ids